app/assets/javascripts/application/messages.js

```diff
@@ -1,7 +1,6 @@
 var mumuki = mumuki || {};
```

Comment:
We should remove this line since we are assuming mumuki object is already initialized.